security/src/test/java/uk/ac/ox/ndph/mts/security/authorisation/AuthorisationServiceTests.java

```diff
@@ -33,19 +36,36 @@
     @Mock
     private SecurityContextUtil securityContextUtil;
 
+
     private AuthorisationService authorisationService;
 
+    private final String managedIdentity = "999";
+
     @BeforeEach
     void setup() {
         this.authorisationService = new AuthorisationService(securityContextUtil,
                 practitionerServiceClient,
                 roleServiceClient);
+        //Set env var
+        ReflectionTestUtils.setField(authorisationService, "managedIdentity", managedIdentity);
     }
 
+    @Test
+    void TestBypassAuthorise_WhenUser_ManagedIdentity(){
+        //Arrange
+        String userId = managedIdentity;
+        when(securityContextUtil.getUserId()).thenReturn(userId);
+
+        //Act
+        //Assert
+        assertTrue(authorisationService.authorise("some-permission"));
+    }
+
+
     @Test
     void TestAuthorise_WithInvalidToken_ReturnsFalse(){
         //Arrange
-        when(securityContextUtil.getUserId()).thenThrow(new AuthorisationException("Any exception during getting user from token"));
+        when(securityContextUtil.getUserId()).thenThrow(new AuthenticationException("Any exception during getting user from token"));
```

Comment:
I know you didn't add this, but seems like a redundant test to me - if the token is invalid how would we get passed the authentication of spring security...
This also makes the AuthenticationException (or AuthZ) class kind of mute.
@LizaShak?

Reply:
I think Liza may have added it to ensure that Spring return a 403 due to forbidden access rules we throw an authenticationException rather than a generic Spring exception. Ill leave this in for the time being as the change/discussion is outside the scope of ARTS-353.